tensorflow_datasets/structured/wisconsin.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wisconsin Breast Cancer dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+DATA_URL = "https://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin"
```

Comment:
The link should be to the .data file, as that is the only way DownloadManager will be able to parse the file. See the implementation of another UCI dataset, the IRIS dataset [here ](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/structured/iris.py)for better understanding.